libs/dataset-explorer/src/lib/FeatureBalanceMeasuresChart.tsx

```diff
@@ -176,6 +179,8 @@ export class FeatureBalanceMeasuresChart extends React.PureComponent<
 
         {/* Renders the description of the chosen measure */}
         <Stack.Item>
+          {/* The description is a paragraph with the format "<measure name bolded> <measure desc>".
+              Because the measure desc wraps to a new line, Label cannot be used for the measure name. */}
           <Text style={{ fontWeight: 600 }}>{selectedMeasure} </Text>
```

Comment:
In this case, I think you can move the fontWeight to a styles file as well.  It still seems strange to me that we can't change label to not wrap for this scenario.